Tests/XMLCoderTests/AdvancedFeatures/ErrorContextTest.swift

```diff
@@ -31,26 +31,34 @@ final class ErrorContextTest: XCTestCase {
                 return
             }
 
-            #if os(Linux) && swift(<5.4)
-            // XML Parser returns a different column on Linux
-            // https://bugs.swift.org/browse/SR-11192
-            XCTAssertEqual(ctx.debugDescription, """
+            let column2 = """
+            \(underlying.localizedDescription) \
+            at line 1, column 2:
+            `<blah `
+            """
+            let column7 = """
             \(underlying.localizedDescription) \
             at line 1, column 7:
             `ah //>`
-            """)
-            #elseif os(Windows) || os(Linux)
-            XCTAssertEqual(ctx.debugDescription, """
+            """
+            let column10 = """
             \(underlying.localizedDescription) \
             at line 1, column 10:
             `//>`
-            """)
+            """
+
+            #if os(Linux) && swift(<5.4)
+            // XML Parser returns a different column on Linux and iOS 16+
+            // https://bugs.swift.org/browse/SR-11192
+            XCTAssertEqual(ctx.debugDescription, column7)
+            #elseif os(Windows) || os(Linux)
+            XCTAssertEqual(ctx.debugDescription, column10)
             #else
-            XCTAssertEqual(ctx.debugDescription, """
-            \(underlying.localizedDescription) \
-            at line 1, column 2:
-            `<blah `
-            """)
+            if #available(iOS 16.0, tvOS 16.0, macOS 13.0, *) {
+                XCTAssertEqual(ctx.debugDescription, column7)
```

Comment:
Do we (want to) guarantee error output from Foundation? If so, this PR alone seems to indicate we cannot.

Reply:
This output seemed to be pretty stable up to this point, and I haven't found a different way to test the error context functionality end-to-end without being exposed to these caveats with how Foundation handle this.